src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -191,141 +205,144 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
+                        EntityDetailsListDataConfigurationType.ENTITY_SEARCH,
+                      query: {
+                        queryTerm: searchTerms,
+                      },
+                    }
+              }
+              showVersionSelection={showVersionSelection}
+              selectColumnType={selectMultiple ? 'checkbox' : 'radio'}
+              selected={selectedEntities}
+              includeTypes={showTypes}
+              selectableTypes={selectableTypes}
+              toggleSelection={toggleSelection}
             />
-          </div>
-        </div>
-        {/* We have a separate Details component for search in order to preserve state in the other component between searches */}
-        {searchActive && (
-          <EntityDetailsList
-            sessionToken={sessionToken}
-            configuration={
-              searchByIdResults && searchByIdResults.length > 0
-                ? {
-                    type: EntityDetailsListDataConfigurationType.HEADER_LIST,
-                    headerList: searchByIdResults,
-                  }
-                : {
-                    type: EntityDetailsListDataConfigurationType.ENTITY_SEARCH,
-                    query: {
-                      queryTerm: searchTerms,
-                    },
-                  }
-            }
-            showVersionSelection={showVersionSelection}
-            selectColumnType={selectMultiple ? 'checkbox' : 'radio'}
-            selected={selectedEntities}
-            includeTypes={showTypes}
-            selectableTypes={selectableTypes}
-            toggleSelection={toggleSelection}
-          />
-        )}
-        {
-          <div style={searchActive ? { display: 'none' } : {}}>
-            <div className="EntityViewReflexContainer">
-              <SizeMe>
-                {({ size }) => (
-                  <ReflexContainer
-                    key={(!!size.width).toString()}
-                    orientation="vertical"
-                    windowResizeAware
-                  >
-                    <ReflexElement minSize={200} size={350}>
-                      <TreeView
-                        sessionToken={sessionToken}
-                        setDetailsViewConfiguration={setConfigFromTreeView}
-                        showFakeRootNode={true}
-                        showDropdown={true}
-                        initialContainer={initialContainerId}
-                      ></TreeView>
-                    </ReflexElement>
-                    <ReflexSplitter></ReflexSplitter>
-                    <ReflexElement minSize={400}>
-                      {configFromTreeView && (
-                        <EntityDetailsList
+          )}
+          {
+            <div style={searchActive ? { display: 'none' } : {}}>
+              <div className="EntityViewReflexContainer">
+                <SizeMe>
+                  {({ size }) => (
+                    <ReflexContainer
+                      key={(!!size.width).toString()}
```

Comment:
I don't think I need windowResizeAware, but I need to look into this further. I force the rerender with SizeMe to try to prevent a bug where the ReflexContainer can't determine its parent's height or width, which causes it to render a blank component. This happens often in SWC locally...I think partially because it loads in a modal but I'm not totally sure. It's been a very difficult issue to fix reliably.